userName/src/main/java/com/example/userName/UserInfo.java

```diff
@@ -0,0 +1,24 @@
+package com.example.userName;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserInfo {
+
+    public String service(String name) {
+        return switch (name) {
+            case "sato" -> "佐藤太郎";
+            case "yamada" -> "山田花子";
+            case "ito" -> "伊藤純";
+            default -> "IDが入力されていない";
+        };
+    }
+    public int userBirthDays(String Birthdays){
```

Comment:
引数に指定したnameから該当の誕生日をintで返すということだと思うのでメソッド名は下記のようにしたらいいと思いました。
```suggestion
    public int getUserBirthdayByName(String name){
```

Reply:
わかりました。